src/redux/features/observations/utils.ts

```diff
@@ -166,8 +166,9 @@ export const mergeObservations = (
 
 const StatusWeights: Record<MonitoringPlotStatus, number> = {
   Completed: 1,
-  InProgress: 2,
-  Outstanding: 3,
+  Claimed: 2,
+  Unclaimed: 3,
+  'Not Observed': 4,
```

Comment:
"Claimed" and "unclaimed" make sense to me, I am not sure about "not observed" though. Maybe that is in the PRD?

Reply:
Couldn't find it. Maybe we should merge it like this and maybe change later if needed?